doc/langref.html.in

```diff
@@ -7031,6 +7031,12 @@ fn readFile(allocator: *Allocator, filename: []const u8) ![]u8 {
                   If no overflow or underflow occurs, returns {#syntax#}false{#endsyntax#}.
       </p>
       {#header_close#}
+      {#header_open|@addWithSaturation#}
+      <pre>{#syntax#}@addWithSaturation(a: T, b: T) T{#endsyntax#}</pre>
+      <p>
+      Returns {#syntax#}a + b{#endsyntax#}. The result will be clamped between the type maximum and minimum.
+      </p>
+      {#header_close#}
```

Comment:
Please also mention that the builtin is temporary, link to https://github.com/ziglang/zig/issues/1284 and mention that a future improvement will change it from a builtin to syntax.